Libraries/Components/TextInput/TextInput.js

```diff
@@ -378,6 +382,15 @@ const TextInput = React.createClass({
      */
     selectionState: PropTypes.instanceOf(DocumentSelectionState),
     /**
+     * The start and end of the text input's selection. Set start and end to
+     * the same value to position the cursor.
+     * @platform ios
+     */
+    selection: PropTypes.shape({
+      start: PropTypes.number.isRequired,
+      end: PropTypes.number.isRequired
```

Comment:
This doesn't block the PR from landing but we could maybe make this more flexible, so that if you specify only a `start` (I think this is common when you want to just reposition the cursor but not select anything) then the TextInput will take care of setting the end to be equal to start for you before passing to native (or we could handle on native side).